bin/upload-staticfiles.sh

```diff
@@ -4,7 +4,11 @@ set -exo pipefail
 
 source docker/bin/set_git_env_vars.sh
 
-if [[ "$BRANCH_NAME" != "prod" ]]; then
+if [[ "$BRANCH_NAME" == "prod" ]]; then
+    BUCKETS=(stage prod)
+elif [[ "$BRANCH_NAME" == "master" ]]; then
+    BUCKETS=(dev)
```

Comment:
What would be the consequences of pushing to dev and stage on the master branch and only updating the prod bucket on push to prod?

Reply:
We will be doing that in a future PR when we implement the new CI pipeline we discussed at Mozlando. This plus a number of other things about the deployment scripts will need to change.